Mage.Client/src/main/java/mage/client/dialog/TableWaitingDialog.java

```diff
@@ -450,8 +451,10 @@ private int getPlayersCount(TableView tableView) {
         int playerCount = 0;
         if (tableView != null) {
             for (SeatView seatView : tableView.getSeats()) {
-                if (seatView.getPlayerId() != null && seatView.getPlayerType().equals("Human")) {
-                    playerCount++;
+                if (seatView.getPlayerId() != null) {
+                    if (seatView.getPlayerType().toString().equals("Human")) {
```

Comment:
Will there be other places where these things have started failing? 
What I found (with equals.*"Human") was:

" mage\Mage.Client\src\main\java\mage\client\table\TablePlayerPanel.java +++++++"
        if (!this.cbPlayerType.getSelectedItem().equals("Human")) {
" mage\Mage.Client\src\main\java\mage\client\table\TournamentPlayerPanel.java +++++++"
        if (!this.cbPlayerType.getSelectedItem().equals("Human")) {
" mage\Mage.Client\src\main\java\mage\client\dialog\NewTournamentDialog.java +++++++"
                if (!player.getPlayerType().getSelectedItem().toString().equals("Human")) {
" mage\Mage.Client\src\main\java\mage\client\dialog\TableWaitingDialog.java +"
                if (seatView.getPlayerId() != null && seatView.getPlayerType().equals("Human")) {